docs/root/intro/version_history.rst

```diff
@@ -5,6 +5,7 @@ Version history
 ===============
 * stats: added :ref:`stats_matcher <envoy_api_field_config.metrics.v2.StatsConfig.stats_matcher>` to the bootstrap config for granular control of stat instantiation.
 * fault: removed integer percentage support.
+* router: added ability to configure arbitrary :ref:`retriable status codes. <envoy_api_field_route.RouteAction.RetryPolicy.retriable_status_codes>`
```

Comment:
I think you have a merge conflict anyway in here, so do you mind just fixing all of this to be alpha order? I think we have had some merge issues. Sorry for the trouble!